### PR TITLE
Fix bug 842092 - Add career listings feed.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

File: careers/careers/feeds.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        return get_all_categories()
+
+    def items(self):
+        return get_all_positions()
+
+    def item_title(self, item):
+        return item.title
+
+    def item_description(self, item):
+        return item.description
```
Comment (jezdez) [inline careers/careers/feeds.py]: I'm not sure if this needs to be marked safe or not, can you shed some light on this?

**Comment by jezdez** [comment]
Umm, when I run the tests locally the test doesn't fail. Weird. I run `docker-compose run web ./manage.py test`. Is that the correct command?

**Comment by glogiotatidis** [comment]
> Umm, when I run the tests locally the test doesn't fail. Weird. I run docker-compose run web ./manage.py test. Is that the correct command?

Yes, I landed #159 to disable ssl redirects on travis. Your tests are fine!

File: careers/careers/feeds.py (hunk 1 above)
Comment (glogiotatidis) [inline careers/careers/feeds.py]: According to the RSS spec you need to escape html in `description` so this doesn't need safe. Firefox displays the feed correctly.

File: careers/careers/feeds.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    def item_categories(self, item):
+        categories = []
+        if item.category:
+            categories.append(item.category.name)
+        if item.location_filter:
+            categories.append(item.location_filter)
```
Comment (glogiotatidis) [inline careers/careers/feeds.py]: Hmmm, `location_filter` is usually `All` which in the feed doesn't mean much. Maybe we translate `All` to `Worldwide`?

Comment (glogiotatidis) [inline careers/careers/feeds.py]: Or we add `Location:` so that `categories.append('Location: {}'.format(item.location_filter))?

**Comment by glogiotatidis** [comment]
Also please rebase of master to fix the tests.

**Comment by glogiotatidis** [comment]
Thanks for this @jezdez ! :+1:  

Just a nit with the location category, other than that great work!

File: careers/careers/feeds.py (hunk 2 above)
Comment (jezdez) [inline careers/careers/feeds.py]: I think I like "Worldwide" better than "Location: All" since I'd be curious what "All" locations are.

**Comment by jezdez** [comment]
@glogiotatidis Done!

**Comment by glogiotatidis** [comment]
Thanks @jezdez :) :cake:

**Pull request merged**